pennylane/devices/tests/conftest.py

```diff
@@ -104,7 +104,7 @@ def _device(wires):
             )
 
         capabilities = dev.capabilities()
-        if "model" not in capabilities or not capabilities["model"] == "qubit":
+        if capabilities.get("model", None) != "qubit":

```

Comment:
Just thinking out-loud here: if we had capability validation in PL-core, we could avoid devices with invalid capabilities being loaded in the first place. E.g., `qml.device("my.device")` would raise an error if `MyDevice.capabilities()["model"] not in ("qubit", "cv")`. This might be a good thing, since it avoids PL loading devices that could lead to unexpected/broken behaviour.

There are two places we could put a validation check:

* `qml.device`

* The QNode itself

Reply:
I put it on to my list!